topscroll/topscroll.js

```diff
@@ -14,10 +14,19 @@ var pixelUrl3 = ('#{PIXEL_URL3}' === defaultPixelUrl) ? '' : '#{PIXEL_URL3}';
 var receiver = window.parent.document.getElementById(document.body.id);
 if (receiver) {
   // calculate height
+  // if an offsetElement is found it will be used instead of the menu height.
   // TODO: don't rely on access to native DOM-elements
-  var menuElements = '.topbar, #sitehead, .masthead';
-  var menuHeight = window.top.document.querySelector(menuElements) ? window.top.document.querySelector(menuElements).offsetHeight : 185;
-  var creativeHeight = window.parent.document.documentElement.clientHeight - menuHeight;
+  var offsetElements = window.top.document.querySelector('#fnTicker');
+  var menuElements = window.top.document.querySelector('.topbar, #sitehead, .masthead');
+  var whitespace = 185;
```

Comment:
@tcarlsen Hvorfor er whitespace default 185?

Reply:
det den default værdi `menuHeight` havde før. så det var for at beholde den:

```
var menuHeight = window.top.document.querySelector(menuElements) ? window.top.document.querySelector(menuElements).offsetHeight : 185;
```

Reply:
Arh yes - skal lige igang igen efter ferien 😃